### PR TITLE
feat(rpc): update rpc methods

### LINKED ISSUE #471: Outdated RPC methods
**Is your feature request related to a problem? Please describe.**

CKB [0.106.0](https://github.com/nervosnetwork/ckb/issues/3710) has been released, there are [some changes](https://github.com/nervosnetwork/ckb/issues/3726) Lumos should care about

**Describe the solution you'd like**

-  `get_block_filter`
-  `get_transaction_and_witness_proof`
-  `verify_transaction_and_witness_proof`
-  `get_fork_block`
-  `get_block_median_time`
-  `estimate_cycles`
-  `get_fee_rate_statics`
-  `get_fee_rate_statistics`

### PR DESCRIPTION
**Pull request by IronLu233**
# Description
This PR add these new methods on RPC:
- get_block_filter
- get_transaction_and_witness_proof
- verify_transaction_and_witness_proof
- get_fork_block
- get_block_median_time
- estimate_cycles
- get_fee_rate_statics
- get_fee_rate_statistics

Fixes #471 
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Website
- [ ] Example
- [ ] Other

# How Has This Been Tested?

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/rpc/__tests__/ckb-rpc.test.js [hunk 1]
```diff
@@ -1007,581 +1104,1046 @@ describe('Test with mock', () => {
...
-          'passthrough',
+          "passthrough",
+        ],
+      });
+      expect(res).toEqual(
+        "0xa0ef4eb5f4ceeb08a4c8524d84c5da95dce2f608e0ca2ec8091191b0f330c6e3"
+      );
+    });
+
+    it.each([
```
Comment (IronLu233) [inline packages/rpc/__tests__/ckb-rpc.test.js:1628]: The new unit tests start here.

File: packages/rpc/__tests__/ckb-rpc.test.js [hunk 2]
```diff
@@ -1007,581 +1104,1046 @@ describe('Test with mock', () => {
...
+          "\n",
+          JSON.stringify({ id, jsonrpc: "2.0", ...expectedParams })
+        );
+        expect(axiosMock.mock.calls[0][0].data).toEqual({
+          id,
+          jsonrpc: "2.0",
+          ...expectedParams,
+        });
+      }
+    );
```
Comment (IronLu233) [inline packages/rpc/__tests__/ckb-rpc.test.js:2044]: The new unit tests end here.

**Review by IronLu233 — COMMENTED** [review]
Please use `
Hide whitespace` for better review experience.
image

**Review by homura — APPROVED** [review]
LGTM

**Pull request merged**